docs/decision-logs/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: Overview
+title: Overview - Decision Logs
+description: Topaz Decision Logs - Overview
+---
+
+# Introduction
+
+Decision logs are records of authorization decisions Topaz makes. Decisions are made by the Topaz [is](docs/authorizer-guide/is) API, 
```

Comment:
link should probably be [is](/docs/authorizer-guide/is)  (notice the leading slash)